modules/graph_ql/main.tf

```diff
@@ -1,4 +1,133 @@
-data "aws_iam_policy_document" "graphql_task_execution_policy_document" {
+# Local Variables
+locals {
+  graphql_port = 5000
```

Comment:
A comment here may be good that lets us know that it is used for the container, host, and ALB ports.

Reply:
Added